cmake/onnxruntime_python.cmake

```diff
@@ -96,11 +70,32 @@ target_link_libraries(onnxruntime_pybind11_state PRIVATE
     ${PROVIDERS_ACL}
     ${PROVIDERS_ARMNN}
     ${PROVIDERS_ROCM}
-    onnxruntime_optimizer
```

Comment:
Do you have a strong reason of removing onnxruntime_optimizer from here? Why onnxruntime_optimizer blocks you using pytorch?

Reply:
onnxruntime_optimizer is not removed, it is put below. It is found onnxruntime_framework reference some code in onnxruntime_optimizer , then if we put it ahead, when building onnxruntime_framework , it is reported some optimizer is not found.

Reply:
let me try not reordering as much as possible.